renderers/liquidcrystal/tcMenuLiquidCrystal.h

```diff
@@ -35,13 +35,15 @@ class LiquidCrystalRenderer : public BaseMenuRenderer {
 	uint8_t editChar;
     bool drewTitleThisTime;
     bool titleRequired;
+    char title[20];
 public:
 
 	LiquidCrystalRenderer(LiquidCrystal& lcd, uint8_t dimX, uint8_t dimY);
 	virtual ~LiquidCrystalRenderer();
 	void render() override;
     void initialise() override;
     void setTitleRequired(bool titleRequired) { this->titleRequired = titleRequired; }
+    void setTitle(const char* newTitle) { strcpy(this->title, newTitle); }
```

Comment:
please can you use the 'n' version of the method to make sure we don't copy much into the title array. Also set the title[sizeof(title)-1] = 0; to ensure it's properly terminated.